File: app/src/main/java/com/bmco/cratesiounofficial/interfaces/OnSummaryChangeListener.java
package com.bmco.cratesiounofficial.interfaces;

import com.bmco.cratesiounofficial.models.Summary;

/**
 * Created by bertuswisman on 26/05/2017.
 */

public interface OnSummaryChangeListener {
    void summary(Summary summary);
    void downloadStarted();
}

File: app/src/main/java/com/bmco/cratesiounofficial/interfaces/OnSummaryChangeListener.kt
package com.bmco.cratesiounofficial.interfaces

import com.bmco.cratesiounofficial.models.Summary

/**
 * Created by bertuswisman on 26/05/2017.
 */

interface OnSummaryChangeListener {
    fun summary(summary: Summary)
    fun downloadStarted()
}
